tests/configs/test_learning_curve_with_objectives.template.cfg

```diff
@@ -9,6 +9,6 @@ suffix=.jsonlines
 
 [Tuning]
 grid_search=false
-objectives=['accuracy', 'f1_score_micro']
 
 [Output]
+metrics=['accuracy', 'f1_score_micro']
```

Comment:
I think you need to rename the template file since it no longer contains objectives.

Reply:
this template was duplicate of another, so it is now deleted.